docs/how-to/configure-base-image.md

```diff
@@ -0,0 +1,24 @@
+# How to configure base-image
+
+To use different Ubuntu OS base, you can use the `base-image` configuration option.
```

Comment:
For my information, what's the default Ubuntu OS base? 

I'm asking because the phrase "To use different Ubuntu OS base" implies that there is a default option, but this guide does not mention what the default is. I would suggest adding that context in as the first sentence:

By default, `github-runner-image-builder-operator` uses the XYZ Ubuntu OS base. To choose a different base, you can use the `base-image` configuration option.